packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -242,7 +245,7 @@ const TagPage = ({classes}: {
   }
 
   const htmlWithAnchors = tag.tableOfContents?.html || tag.description?.html;
-  const description = (truncated && !tag.wikiOnly)
+  const description = (truncated && !tag.wikiOnly && !isEAForum)
```

Comment:
Curious why this was changed? For longer tags I can imagine this cutting against the sub-forum vibe we're moving toward with topics, by pushing down the posts section. Probably not a big deal either way—a brief attempt to query for tags by length didn't work for reasons I haven't pinned down, but definitely most of them seem short.

Reply:
I'm mostly deferring to Pablo [here ](https://eaforummoderators.slack.com/archives/C027Z3794F3/p1649108997863579) and [here](https://cea-core.slack.com/archives/C01Q279LKCK/p1649185592391149).

I agree that if/when we do sub forums we might have to rethink this